src/components/BooksList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BooksList = ({ books }) => (
+  <div>
+    <h2>BooksList</h2>
+    <table>
+      <thead>
+        <tr>
+          <th>Book ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </thead>
+      <tbody>
+        {books.map(book => (
+          <Book key={book.id} book={book} />
+        ))}
+      </tbody>
+    </table>
+  </div>
+);
+
+BooksList.propTypes = {
+  books: PropTypes.arrayOf(Book.propTypes).isRequired,
```

Comment:
Using [PropTypes.shape](https://dev.to/cesareferrari/how-to-specify-the-shape-of-an-object-with-proptypes-3c56) here will allow you to set a type check that is more accurate.